db/migrations/dedupe_github_repo_owner.py

```diff
@@ -0,0 +1,61 @@
+"""Remove case-sensitive duplicates in GitHub repository tables"""
+
+import rethinkdb as r
+
+import db.plugins
+import db.util
+
+r_conn = db.util.r_conn
+
+
+if __name__ == '__main__':
+    table = 'plugin_github_repos'
+
+    print 'Removing duplicate rows in %s' % table
+
+    updated = 0
+    deleted = 0
+
+    query = r.table(table)
+
+    # Group by the normalized GitHub path
+    query = query.group([
+        r.row['owner'].downcase(),
+        r.row['repo_name'].downcase()])
+
+    # Get the most recently scraped row in each group first
+    query = query.order_by(r.desc('last_scaped_at'))
+
+    grouped_repos = query.run(r_conn())
+
+    for owner_repo, repos in grouped_repos.iteritems():
+
+        print '\nRepo with GitHub path %s occurs %s times' % (
+                owner_repo,
+                len(repos))
+
+        # Use the most recently scraped row as the canonical row
+        canonical = repos.pop()
```

Comment:
I suspect this is not the case, but I'll ask anyway so someone more familiar with the data model can make the correct call: are there valid instances where `repos` is an empty list? If so, this will raise an `IndexError`, which should be handled.